tests/dummy/app/index.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Initial page title - removed by addon initializer</title>
+    <!-- <title>Initial page title - removed by addon initializer</title> -->
```

Comment:
I think apps will always have to remove this, we should remove it for them in their blueprint. And in the RFC that suggests adding ember-page-title to all new apps we should explicitly state that this will be removed in favor of actually invoking `{{title}}` in the `application.hbs`.

Reply:
FWIW, I'm pretty sure that this was _already_ a constraint and not something new. Users of existing ember-page-title releases that are using ember-cli-fastboot along with ember-cli-head would have had to remove `<title>` from `app/index.html`.

Reply:
Wait, how we can this? If app/index.html has no `<title>` then search engines index pages without titles out of the box. All non fastboot apps that today set meaningful static title for search engines will have conflicting setup out of box.

Reply:
I suppose we could have ember-cli-fastboot's blueprint remove it? But for the most part, isn't it safe to assume the search engine will run our JS nowadays?

Reply:
Not sure about. 

https://html.spec.whatwg.org/multipage/semantics.html#the-title-element - it does not say directly title is mandatory but seems strongly recommended.

https://validator.w3.org/ - from this tool perspective page without title is not valid HTML page.

Reply:
I made tests assertions smarter to cope with 2 title tags being present for Fastboot tests with knowledge that in real life we'll ask people or make ember-cli-fastboot blueprint remove title from index.html.